processing/src/main/java/org/apache/druid/segment/join/PostJoinCursor.java

```diff
@@ -99,15 +108,15 @@ public Filter getPostJoinFilter()
   @Override
   public void advance()
   {
-    advanceUninterruptibly();
-    BaseQuery.checkInterrupted();
+    baseCursor.advance();
```

Comment:
Will be good to put some comments as in why this change is being done in the code for a future developer to follow.

Reply:
Added comments.